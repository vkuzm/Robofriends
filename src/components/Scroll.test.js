import React from 'react';
import Scroll from './Scroll';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

it('Scroll component renders without crashing', () => {
	const shallowComponent = toJson(shallow(<Scroll />));
  expect(shallowComponent).toMatchSnapshot();
});