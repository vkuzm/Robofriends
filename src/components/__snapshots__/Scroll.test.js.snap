// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scroll component renders without crashing 1`] = `
<div
  style={
    Object {
      "border": "1px solid black",
      "height": "500px",
      "overflowY": "scroll",
    }
  }
/>
`;
